qemu/tests/cfg/virtio_chardev_trace.cfg

```diff
@@ -0,0 +1,6 @@
+- virtio_chardev_trace:
+    only RHEL
```

Comment:
should be `only Linux`

Reply:
Hi but on bugzilla and polarion it said it's only for RHEL

Reply:
Yeah, `only Linux` is the KEY VALUE in automation, you can consider it as RHEL

Reply:
> Hi but on bugzilla and polarion it said it's only for RHEL

And there is no handle for value RHEL so we even can't run this auto case.

Reply:
Hi but why can I ran this case?

Reply:
> Hi but why can I ran this case?

when I add the new line at the end of the file it works well as well.

Reply:
Ok I updated the code